spec/system/consumer/checkout/details_spec.rb

```diff
@@ -396,6 +396,12 @@
           expect(page).to have_field "order_bill_address_attributes_zipcode", with: "TST01"
         end
       end
+
+      describe "show page title as Checkout Details - Open Food Network" do
+        it "should display title as Checkout Details - Open Food Network" do
+          page.has_title? "Checkout Details - Open Food Network"
+        end
+      end
```

Comment:
This test case doesn't fail if the title isn't correct. If you want to validate a value, you need to use `expect(page).to ...`.

Reply:
@mkllnk you already pushed the changes.
Thank you